### PR TITLE
feat: Bump mermaid version 10.9.1 to 11.3.0

### PR DESCRIPTION
**Pull request by MaciejKucia**
Simply updates mermaid dependency to enable glorious new diagramming features

### CODE REVIEW

File: sampledocs/docs/index.md [hunk 1]
```diff
@@ -236,3 +234,18 @@ zenuml
     Alice->John: See you later!
 ```
 
+### Architecture Beta
+
+```
+architecture-beta
```
Comment (johanneswuerbach) [inline sampledocs/docs/index.md:240]: This requires also an update of the diagram detection code https://github.com/johanneswuerbach/backstage-plugin-techdocs-addon-mermaid/blob/main/src/Mermaid/hooks.ts#L18, could you add this?

Comment (MaciejKucia) [reply sampledocs/docs/index.md:240]: yes

**Comment by karlhaworth** [comment]
@MaciejKucia and @johanneswuerbach I look forward to this PR being merged! We wish to use the `architecture-beta` chart!

**Comment by johanneswuerbach** [comment]
Thanks @MaciejKucia for the PR, I took the liberty to finish this https://github.com/johanneswuerbach/backstage-plugin-techdocs-addon-mermaid/pull/56 and released it as `backstage-plugin-techdocs-addon-mermaid@0.14.0`

**Comment by MaciejKucia** [comment]
Thanks, could not figure out how to test this properly.

**Pull request closed without merging**